piff/input.py

```diff
@@ -575,7 +599,7 @@ def _makeStarsFromImage(image_kwargs, cat_kwargs, wcs, chipnum,
                                 "Using smaller than the full stamp size: %s", x, y, bounds)
                 else:
                     logger.warning("Star at position %f,%f overlaps the edge of the image.  "
-                                    "Skipping this star.", x, y)
+                                    "Skipping this star.", x, y)                    
```

Comment:
Not a big deal, but there's some gratuitous white space added here.  (@jmeyers314 has trained me to avoid this...)